### PR TITLE
better error handling

### PR DESCRIPTION
**Pull request by HubertBalcerzak**

### CODE REVIEW

File: src/main/kotlin/me/hubertus248/deployer/applications/exception/ApplicationCorruptedException.kt [hunk 1]
```diff
@@ -0,0 +1,6 @@
+package me.hubertus248.deployer.applications.exception
+
+import me.hubertus248.deployer.common.exception.ApplicationException
+import org.springframework.http.HttpStatus
+
+class ApplicationCorruptedException(message: String) : ApplicationException(HttpStatus.INTERNAL_SERVER_ERROR, message)
```
Comment (KocproZ) [inline src/main/kotlin/me/hubertus248/deployer/applications/exception/ApplicationCorruptedException.kt:9]: Some documentation would be nice (What does it mean? When is it thrown?)

File: src/main/kotlin/me/hubertus248/deployer/applications/controller/ApplicationController.kt [hunk 2]
```diff
@@ -2,25 +2,18 @@ package me.hubertus248.deployer.applications.controller
...
 import org.springframework.data.domain.Pageable
 import org.springframework.data.web.PageableDefault
 import org.springframework.http.HttpStatus
 import org.springframework.security.core.Authentication
 import org.springframework.validation.annotation.Validated
 import org.springframework.web.bind.annotation.*
-import org.springframework.web.servlet.view.RedirectView
 import java.security.Principal
 
 //todo write tests
```
Comment (KocproZ) [inline src/main/kotlin/me/hubertus248/deployer/applications/controller/ApplicationController.kt:19]: :)

**Review by KocproZ — APPROVED** [review]

File: src/main/kotlin/me/hubertus248/deployer/applications/controller/ApplicationController.kt (hunk 2 above)
Comment (HubertBalcerzak) [reply src/main/kotlin/me/hubertus248/deployer/applications/controller/ApplicationController.kt:19]: :)

File: src/main/kotlin/me/hubertus248/deployer/applications/exception/ApplicationCorruptedException.kt (hunk 1 above)
Comment (HubertBalcerzak) [reply src/main/kotlin/me/hubertus248/deployer/applications/exception/ApplicationCorruptedException.kt:9]: good idea

**Pull request merged**